lib/fizzy/parser.hpp

```diff
@@ -140,4 +140,22 @@ struct parser<Code>
         return result;
     }
 };
+
+template <>
+struct parser<Data>
+{
+    parser_result<Data> operator()(const uint8_t* pos)
+    {
+        MemIdx memidx;
+        std::tie(memidx, pos) = leb128u_decode<uint32_t>(pos);
+
+        unsigned offset;
+        std::tie(offset, pos) = leb128u_decode<uint32_t>(pos);
+
+        std::vector<uint8_t> init;
+        std::tie(init, pos) = parser<std::vector<uint8_t>>{}(pos);
```

Comment:
I don't think this is correct because initializer is a constant expression (sequence of instructions).
I think it would be good to reuse here expression parser of `Global`.

Reply:
The initialiser is a byte array and not an expression.

Reply:
Sorry, I meant this not for initializer but for the offset

Reply:
Hm, the `offset` might be an expression.

Reply:
Fixed.